v1.0/explain.md

```diff
@@ -4,10 +4,24 @@ summary: The EXPLAIN statement provides information you can use to optimize SQL
 toc: false
 ---
 
-The `EXPLAIN` [statement](sql-statements.html) returns CockroachDB's query plan to execute [`DELETE`](delete.html), [`INSERT`](insert.html), [`SELECT`](select.html) or [`UPDATE`](update.html) statements. You can then use this information to optimize those queries.
+The `EXPLAIN` [statement](sql-statements.html) returns CockroachDB's query plan to execute the [explainable statements](#explainable-statments). You can then use this information to optimize those queries.
```

Comment:
nit: `to execute the explainable statements` > `for an explainable statement`.

Also, I'd tweak the second sentence a bit: `You can then use this information to optimize the query.`

Reply:
Please apply this change to other versions of the page as well.

Reply:
done